core/src/main/java/org/apache/calcite/rel/metadata/BuiltInMetadata.java

```diff
@@ -83,13 +83,18 @@ public interface UniqueKeys extends Metadata {
      * represented as an {@link org.apache.calcite.util.ImmutableBitSet}, where
      * each bit position represents a 0-based output column ordinal.
      *
+     * <p>Note that a key plus other columns is still unique. Therefore a table
+     * with all columns unique has a unique key consisting of the empty set.
+     * This is the case with all single-row tables.
```

Comment:
Not true. The tuple with zero columns has only one possible value. Therefore it is unique only in tables with 0 or 1 rows. (In such tables every combination of columns is unique.)